db-models/listings.sql

```diff
@@ -0,0 +1,14 @@
+-- db-models/listings.sql
+CREATE TABLE listings (
+    id INTEGER PRIMARY KEY AUTOINCREMENT,       -- Auto-incrementing ID
+    author TEXT NOT NULL,                       
+    title TEXT NOT NULL,                     
+    price  NOT NULL,                            -- Floating-point number
+    item_description TEXT NOT NULL,              
+    seller TEXT NOT NULL,                       
```

Comment:
Since we have `seller_id`, we don't need a `seller` field too. We can just get the `user` row associated with that `seller_id` and get their name from there